docs/tools/boot.mdx

```diff
@@ -147,6 +147,13 @@ pyluos-bootloader flash COM3 -b firmware_new.bin
 No matter what problem you encounter during the loading process, you have to power-off / power-on your network to see all the nodes running in bootloader mode. Once you get there, you have to use `pyluos-bootloader detect` / `flash` tools to load applications and make it work fine.
 :::
 
+### ST-Link connected gate
+
+If for the communication of the gate with your computer, you are using the ST-Link protocol, and the flashing process is blocked in the last step of flashing with no specific reason, then you may have come through a specific problem caused by the ST-Link driver, with a specific size of binary.
+:::tip
+To reasure yourself that this is the case, you can try changing the size of your binary by changing some bytes of your code, or you can use an other way to connect your gate, like for example JTAG. 
```

Comment:
I'm not sure Jtag is the more obvious link here we are talking about serial connection. Why don't we propose an FTDI instead?